hvcc/generators/ir2c/static/HvUtils.h

```diff
@@ -273,6 +273,23 @@ static inline hv_int32_t __hv_utils_min_i(hv_int32_t x, hv_int32_t y) { return (
   }
 #endif
 #define hv_min_max_log2(a) __hv_utils_min_max_log2(a)
+#define hv_if_f(a, b, c) ((a) ? (b) : (c))
+#define hv_modf_f(a) fmod(a, 1)
+#define hv_cbrt_f(a) cbrt(a)
+#define hv_copysign_f(a, b) copysign(a, b)
+#define hv_remainder_f(a, b) remainder(a, b)
+#define hv_erf_f(a) erf(a)
+#define hv_erfc_f(a) erfc(a)
+#define hv_expm1_f(a) expm1(a)
+#define hv_finite_f(a) finite(a)
+#define hv_fmod_f(a, b) fmod(a, b)
```

Comment:
I realized there is a difference between implementations for floats and doubles ..

Reply:
oh yes, have you seen #276 ?

Reply:
Ah, we are fragmenting different discussions all over the place now haha.

I've now made a list directly pointing to specific redefined implementations (this also serves as "documentation" to which objects are supported).

What do you think of this approach?

Reply:
Looks good, but it is clear for every function that it is processing float32? If they are template functions, like in <cmath>, explicit casting to (float) for the arguments could be beneficial.

Reply:
From what I can tell all functions ending with `f` operate on floats. The linter in my editor noted the difference.

Ie. `finite()` and `finitef()`

Reply:
hmmm, not sure i understand. The defines resolve to functions without trailing f.
So, following, e.g. https://en.cppreference.com/w/cpp/numeric/math/expm1, we would have to use
`#define hv_expm1_f(a) expm1((float)(a))` to be really sure.
Or with C++23, `#define hv_expm1_f(a) expm1f(a)`.

Reply:
I was considering what we already had here: https://github.com/Wasted-Audio/hvcc/blob/develop/hvcc/generators/ir2c/static/HvUtils.h#L236-L256